packages/cli/test/integration/shared/utils/testServer.ts

```diff
@@ -275,7 +276,11 @@ export const setupTestServer = ({
 						registerController(
 							app,
 							config,
-							new TagsController({ config, externalHooks, repositories }),
+							new TagsController({
```

Comment:
same as mentioned above: if we make `TagsController` a `@Service`, can replace this block with `Container.get(TagsController)`